nextest-runner/src/reporter.rs

```diff
@@ -842,16 +842,17 @@ impl<'a> TestReporterImpl<'a> {
                                     false => self.failure_output,
                                 };
 
-                                if self.final_status_level >= final_status_level {
+                                // Print out the final status line so that status lines are shown
+                                // for tests that e.g. failed due to signals.
+                                if self.final_status_level >= final_status_level
+                                    || test_output_display.is_final()
+                                {
```

Comment:
this diff looks strange, since the lines are all updated in nextest/main, and I thought shouldn't has the diff when I want to merge into *[nextest-rs:main](https://github.com/nextest-rs/nextest) from [jiangying000:main](https://github.com/jiangying000/nextest)*
https://github.com/nextest-rs/nextest/blob/main/nextest-runner/src/reporter.rs#L845-L850

Reply:
The code in nextest-rs:main looks just the same as jiangying000:main, why is the diff ?